confgenerator/logging_receivers.go

```diff
@@ -430,11 +430,12 @@ func (r LoggingReceiverWindowsEventLog) Components(ctx context.Context, tag stri
 	input := []fluentbit.Component{{
 		Kind: "INPUT",
 		Config: map[string]string{
-			"Name":         inputName,
-			"Tag":          tag,
-			"Channels":     strings.Join(r.Channels, ","),
-			"Interval_Sec": "1",
-			"DB":           DBPath(tag),
+			"Name":           inputName,
+			"Tag":            tag,
+			"String_Inserts": "true",
```

Comment:
Nit: can you add a comment referencing that this is a temporary workaround until https://github.com/fluent/fluent-bit/pull/8853 is fixed?